docs/spec_draft.md

```diff
@@ -82,13 +82,70 @@ The specification of an op comprises of the following components (in the order
   * **Constraints** Type constraints on the operand(s), result(s).
   * **Examples** Examples demonstrating the working of the op.
 
+### stablehlo.abs
+
+`stablehlo.abs(operand) -> result`
+
+### Semantics
+
+Performs element-wise absolute value of `operand` tensor and produces a `result`
+tensor. For a complex value `(a, b)`, the absolute value is defined as
+$\sqrt(a^2 + b^2)$.
```

Comment:
How about we remove the formula, since it's defined in calculus textbooks anyway? 

This is similar to other related simplifications that we discussed recently - e.g. a proposal to remove details of how min/max is supposed to work on floats or to remove the formula for tanh. The less text we write, the less mistakes we can make :)

In comparison, I think that the `minimum`/`maximum` semantics for complex numbers is important to keep in our specification, because I don't think this is part of textbook calculus.